src/tasks_downstream.py

```diff
@@ -764,6 +764,103 @@ def down(c, purge=False):
         c.run(cmd, pty=True)
 
 
+def check_make_yaml(filename):
+    yaml_exists = os.path.exists(PROJECT_ROOT / filename)
+    if not yaml_exists:
+        # Create empty yaml file
+        with open(PROJECT_ROOT / filename, "w") as f:
+            f.write("namespaces: []\n")
+
+
+@task(
+    help={
+        "command": "Command to run in the container. "
+        "Options: ['shell', 'logs', 'bash', 'upgradelog', 'pgactivity', 'removens']",
+    }
+)
+def kube(c, command, namespace="none", db="none"):
```

Comment:
```suggestion
def kube(c, command, namespace=None, db=None):
```

Reply:
For some reason I had issues with this saying "No Idea What {namespace} is!"... but I just tried this and it worked this time. Meh